ckanext/data_qld/plugin.py

```diff
@@ -20,23 +37,14 @@
 from .resource_freshness import validation as resource_freshness_validator
 from .resource_freshness.logic.actions import get as resource_freshness_get_actions
 
-try:
-    from ckanext.qa.interfaces import IQA
-    import ckanext.qa.lib as qa_lib
-    import ckanext.qa.tasks as qa_tasks
-    import os
-    qa_present = True
-except ImportError:
-    qa_present = False
-
-
 log = logging.getLogger(__name__)
 
 
-class DataQldPlugin(plugins.SingletonPlugin):
+class DataQldPlugin(CKANHarvester):
```

Comment:
extension?

Reply:
Yeah, that's how the harvester apparently works. I prefer the usual interface-based setup, but c'est la vie.